src/main/java/innovimax/mixthem/operation/LineResult.java

```diff
@@ -0,0 +1,156 @@
+package innovimax.mixthem.operation;
+
+import java.util.EnumSet;
+
+/**
+* <p>Describes the result of an line operation on two files.</p>
+* @author Innovimax
+* @version 1.0
+*/
+public class LineResult {
+    
+    private final EnumSet<ResultType> types;
+    private String line1;
+    private String line2;
+    private String result;
+    
+    /**
+    * Creates a line result.    
+    */
+    public LineResult() {
+        this.types = EnumSet.noneOf(ResultType.class);
+        this.types.add(ResultType._READ_BOTH_FILES);
```

Comment:
why not using EnumSet.of(ResultType._READ_BOTH_FILES) ?

https://docs.oracle.com/javase/9/docs/api/java/util/EnumSet.html#of-E-

Reply:
exact